src/services/users/create-new-user.test.js

```diff
@@ -0,0 +1,60 @@
+const { expect } = require('chai')
+const { faker } = require('@faker-js/faker')
+const db = require('../../db')
+const {fixtures} = require('../../../test/utils')
+const createNewUser = require('./create-new-user')
+
+
+describe('Create a New User', () => {
+	let signupInfo
+	let userInfo = {}
+	
+
+	beforeEach(async () => {
+		userInfo = {
+			firstName: faker.name.firstName(),
+			lastName: faker.name.lastName(),
+			email: faker.internet.email(),
+			password: faker.internet.password()
+		}
+		signupInfo = await fixtures.insertUserInvite()
+	})
+
+	it('should throw an error if the invite is invalid', async () =>
+
+		expect(createNewUser({...userInfo}))
+			.to.be.rejectedWith(
+				'Email is not present in invites list')
+	)
+    
+	it('should throw an error if user already exists', async () =>{
+		const { email } = signupInfo
+		await fixtures.insertUserInvite({email})
+		await fixtures.updateInviteStatus({email})
+		return expect(createNewUser({
+			firstName:userInfo.firstName,
+			lastName:userInfo.lastName, email, 
+			password:userInfo.password}))
+			.to.be.rejectedWith(
+				'User already exists')
+	})
+
+
+	it('should create a new user', async () =>{
+		const { email } = signupInfo
+		await createNewUser({
+			firstName:userInfo.firstName, 
+			lastName:userInfo.lastName, email, 
+			password:userInfo.password})
+
+		const {rows} = await db.query(
+			`SELECT * FROM users
+             WHERE email = $1`,[email] )
+
+		const createdUser = rows[0]
+	
+		return expect(createdUser.firstName).to.equal(userInfo.firstName)
```

Comment:
Okay, cool. I think the test case is fine. We can bring back the `exist` we were using

What I was suggesting will have to be another test case. like `it(should return right data)`

Currently, I think we return `{accessToken, refreshToken, userId}`. If someone  takes out the `refreshToken` from this, the test should fail

so we check the output `createNewUser` function to make sure it is returning the right data

Reply:
@jocrah I have updated it